tests/assembly/asm/inline-asm-avx.rs

```diff
@@ -0,0 +1,26 @@
+// assembly-output: emit-asm
+// compile-flags: --crate-type=lib
+// compile-flags: --target x86_64-unknown-linux-gnu
+// needs-llvm-components: x86
+// force-host
```

Comment:
To @Nilstrieb's comment, I think you can do this as 
```suggestion
// only-x86_64
// ignore-sgx
```
(Why is `sgx` here?  I have no idea.  I just cargo-culted it from <https://github.com/rust-lang/rust/blob/master/tests/assembly/x86_64-floating-point-clamp.rs#L7>.)